portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -136,11 +139,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendError(HttpServletResponse.SC_FORBIDDEN);
     }
 
+    /* Open a connection to blobstore */
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    /* Get a list of all files uploaded to blobstore from this request */
```

Comment:
When reading this comment and the code, the question that pops into my head is

Why are you getting a list of all the files uploaded to blobstore when you want to upload a new file?
My suggestion would be to have the comment here focus on the context of why you are doing this, rather than what it is doing.

Reply:
I've improved this now 👍 https://github.com/thomasogara/step-repository/pull/48/commits/af7b70d388607c242dd6d768de3af0306352c894